Makefile

```diff
@@ -31,8 +31,8 @@ install:
 	install -Dm 755 src/lib/* -t "${DESTDIR}${PREFIX}/share/${pkgname}/lib/"
 
 	# Install icons
-	install -Dm 664 res/icons/"${pkgname}"*.svg -t "${DESTDIR}${PREFIX}/share/icons/hicolor/scalable/apps/"
-	install -Dm 664 res/icons/"${pkgname}"_updates-available*.svg -t "${DESTDIR}${PREFIX}/share/icons/hicolor/scalable/apps/"
+	install -Dm 664 res/icons/"cachy-update"*.svg -t "${DESTDIR}${PREFIX}/share/icons/hicolor/scalable/apps/"
+	install -Dm 664 res/icons/"cachy-update"_updates-available*.svg -t "${DESTDIR}${PREFIX}/share/icons/hicolor/scalable/apps/"
```

Comment:
why not just update pkgname and _pkgname values instead of hardcoding cachy-update string

Reply:
Because only the icon files has been renamed to "cachy-update", all the other files still refers to "arch-update" and needs the pkgname/_pkgname values to remain as-is.

If you want variables such as `pkgname` / `name` to be updated to match "cachy-update" then the whole program must be completely revamped & rebranded to "cachy-update" everywhere (including binary names, man pages names, config file names, etc....).  
While that's doable, @ptr1337 told me that the aim here was only to change the icons and rebrand the "user facing" parts in order to keep the "downstream" changes inside this fork as slight as possible.

Reply:
> Hmm I think it makes it less maintainable, and in case of rebases against upstream would require more work IMHO

Indeed. If you prefer straightforward rebases against upstream, then the **whole** program needs to be rebranded (so not just icons and user facing element, but also binaries, doc, man pages, etc...). As I said, that's doable but it also implies way more work. 
Up to you though :slightly_smiling_face:

Reply:
Sure, I can do that :)